controllers/helpers.go

```diff
@@ -337,6 +337,11 @@ func GetImageByNameOrUUID(ctx context.Context, client *prismclientv3.Client, ima
 	}
 }
 
+func ImageMarkedForDeletion(image *prismclientv3.ImageIntentResponse) bool {
```

Comment:
this doesn't have to be an exported function

Reply:
Fixed in https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/492/commits/0bbdd905ea820b13dc6dbc6846df0da17d46b464